hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/merkle/disk/OnDiskKeySerializer.java

```diff
@@ -45,13 +39,14 @@
  *
  * @param <K>
  */
-public final class OnDiskKeySerializer<K>
-        implements KeySerializer<OnDiskKey<K>>, SelfSerializableSupplier<OnDiskKey<K>> {
-    /** This is a hint for virtual maps, but isn't actually useful. We just pick some size. */
+public final class OnDiskKeySerializer<K> implements KeySerializer<OnDiskKey<K>> {
+    /**
+     * This is a hint for virtual maps, but isn't actually useful. We just pick some size.
```

Comment:
I know this comment isn't a part of this PR, but is copied from elsewhere, but still... This typical size is actually very useful. It defines how often virtual entities are flushed to disk. So please, add a comment (to this field or to getTypicalSerializedSize() method) about revisiting the value in the future based on real mainnet entities.

Reply:
I updated the comment